README.md

```diff
@@ -107,9 +107,9 @@ later.
 
 ## Installing NEST
 
-Generic installation instructions can be found in the
-[INSTALL](https://github.com/nest/nest-simulator/blob/master/INSTALL) file that
-you received in the NEST sources.
+Generic installation instructions can be found in the [NEST
+documentation](https://nest-simulator.org/installation/) on Read the
+Docs.
```

Comment:
Given that we generate the ReadTheDocs doc from material in this repo, I'd include a pointer to the corresponding file here, just in case someone takes a memory stick with NEST sources along to a lonely island without network access ;).